examples/01_encodings.py

```diff
@@ -241,6 +263,9 @@
 # Therefore it can be a faster and very effective alternative, when the supervised
 # learner is built on top of decision trees, which is the case for the |HGBR|.
 
+# The resulting pipeline is identical to the one produced by default by
```

Comment:
This is interpreted as a comment in a code block, if it's on purpose, I'd add it just after the `pipeline=make_pipeline[...]`

Reply:
nope not on purpose :)